src/bin/lpc55/cli.rs

```diff
@@ -95,6 +95,10 @@ pub fn app() -> clap::App<'static, 'static> {
                      .help("filename of file containing settings")
                      .required(true)
                 )
+                .arg(Arg::with_name("OUTPUT")
```

Comment:
I'd prefer not having optional positional arguments - either something is mandatory, then it's positional, or it's optional, then it has a flag.